DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/lacp/test_lacp_routing_over_vrf_and_vlan_device.py

```diff
@@ -0,0 +1,215 @@
+    4. Configure ip addresses in each LAG and configure route 10.1.1.0/24 via 2 vrf's
+    5. Setup one stream with destination IP 10.1.1.1
+    6. Transmit traffic
+    7. Verify traffic received on active bond2 port in vrf0 and not received on vrf1
+    """
+    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 4)
+    if not tgen_dev or not dent_devices:
+        pytest.skip('The testbed does not have enough dent with tgen connections')
+    device = dent_devices[0]
+    dent = device.host_name
+    tg_ports = tgen_dev.links_dict[dent][0]
+    ports = tgen_dev.links_dict[dent][1]
+    bonds = [f'bond_{idx}' for idx in range(1, 4)]
+    tgen_lag_1 = ('LAG_0', [tg_ports[0]])
+    tgen_lag_2 = ('LAG_1', [tg_ports[1]])
+    tgen_lag_3 = ('LAG_2', tg_ports[2:])
+    vrf_1 = 'vrf0'
+    vrf_2 = 'vrf1'
+    vid = 10
+    bridge = 'bridge_0'
+    vlan_device = f'{bridge}.{vid}'
+    rate = 10000
+
+    # 1. Create 3 bonds and a bridge
+    out = await IpLink.add(input_data=[{dent: [{
+            'device': bridge,
+            'type': 'bridge',
+            'vlan_filtering': 1,
+            'vlan_default_pvid': 10,
+            }]}])
+    assert out[0][dent]['rc'] == 0, 'Failed to add bridge'
+
+    out = await IpLink.set(input_data=[{dent: [{'device': bridge, 'operstate': 'up'}]}])
+    assert out[0][dent]['rc'] == 0, 'Failed setting bridge to state up'
+
+    # Bridge to vlan id
+    out = await BridgeVlan.add(input_data=[{dent: [{'device': bridge, 'vid': vid, 'self': True}]}])
+    assert out[0][dent]['rc'] == 0, 'Failed to add vlan on device'
+
+    ####
+    out = await IpLink.add(input_data=[{dent: [{'device': bond, 'type': 'bond', 'mode': '802.3ad'} for bond in bonds]}])
+    assert out[0][dent]['rc'] == 0, 'Failed to add bond'
+
+    out = await IpLink.set(input_data=[{dent: [{'device': bond, 'operstate': 'up'} for bond in bonds]}])
+    assert out[0][dent]['rc'] == 0, 'Failed setting bond to state up'
+
+    # Create 2 VRF tables
+    out = await IpLink.add(input_data=[{dent: [{'dev': vrf_1,  'type': 'vrf', 'table': 10}]}])
+    assert out[0][dent]['rc'] == 0, 'Failed to add vrf'
+
+    out = await IpLink.set(input_data=[{dent: [{'device': vrf_1, 'operstate': 'up'}]}])
+    assert out[0][dent]['rc'] == 0, 'Failed setting vrf to up state'
+
+    out = await IpRoute.add(input_data=[{dent: [{'table': 10, 'type': 'unreachable', 'table_id': 'default'}]}])
+    assert out[0][dent]['rc'] == 0, 'Failed adding route'
+
+    out = await IpLink.add(input_data=[{dent: [{'dev': vrf_2,  'type': 'vrf', 'table': 20}]}])
+    assert out[0][dent]['rc'] == 0, 'Failed to add vrf'
+
+    out = await IpLink.set(input_data=[{dent: [{'device': vrf_2, 'operstate': 'up'}]}])
+    assert out[0][dent]['rc'] == 0, 'Failed setting vrf to up state'
+
+    out = await IpRoute.add(input_data=[{dent: [{'table': 20, 'type': 'unreachable', 'table_id': 'default'}]}])
+    assert out[0][dent]['rc'] == 0, 'Failed adding route'
```

Comment:
why can't you first create two vrfs using one api call? why do you need to separate vrf1 and vrf2 calls?